js-apps/meep-frontend/src/js/containers/exec/network-characteristics-event-pane.js

```diff
@@ -385,9 +398,10 @@ class NetworkCharacteristicsEventPane extends Component {
               onChange={event => {
                 var elem = this.firstElementMatchingType(event.target.value);
                 this.props.updateElement(elem);
-                this.setState({ currentElementType: event.target.value });
+                // this.setState({ currentElementType: event.target.value });
```

Comment:
We should clear the selected element if not matches the selected type.

Reply:
Selected element will automatically get updated now since we are updating the list based on scenario deployed.